### ISSUE TITLE
More Performance 

### ISSUE THREAD

**Issue by initialflame** [alump/LazyLayouts#8]
Thank you for this Addon! It's easy and Awesome! :-)

A nice Feature would be to replace loaded Components if they are not more displayed because of the User has scrolled down. This helps the Browser-Performance if complex layouts with a lot of Compoents are displayed.

**Comment by alump** [comment]
Yes that would be nice feature, but it would limit use cases and kinda break Vaadin layouting logic that this add-on is trying to maintain.
